### PR TITLE
CB-3778-bug-use-oracle-as-internal-db

### PR DESCRIPTION
**Pull request by yagudin10**

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/translate/SQLQueryTranslator.java [hunk 1]
```diff
@@ -84,7 +94,10 @@ public static String translateScript(
             sql.append(element.getText());
             sql.append(scriptDelimiter).append("\n");
         }
-        return sql.toString();
+        if (sql.length() > 0) {
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/translate/SQLQueryTranslator.java]: Why?

File: plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/translate/SQLQueryTranslator.java [hunk 2]
```diff
@@ -222,25 +235,64 @@ protected List<? extends SQLScriptElement> translateStatement(
                     }
                 }
             }
-            if (defChanged) {
-                String newQueryText = SQLFormatUtils.formatSQL(null,
+            if (dialectName.equals(DIALECT_NAME_ORACLE) && !CommonUtils.isEmpty(createTable.getIndexes())) {
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/translate/SQLQueryTranslator.java]: It is a bad idea to use if/else with dialect names. This logic must be implement inside dialect implementation.

**Review by serge-rider — CHANGES_REQUESTED** [review]

File: plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/translate/SQLQueryTranslator.java (hunk 1 above)
Comment (yagudin10) [reply plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/translate/SQLQueryTranslator.java]: removed

File: plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/translate/SQLQueryTranslator.java (hunk 2 above)
Comment (yagudin10) [reply plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/translate/SQLQueryTranslator.java]: fixed

**Review by serge-rider — APPROVED** [review]

**Pull request merged**